website/blog/2021/06/28/why-we-need-Apache-APISIX.md

```diff
@@ -173,6 +173,6 @@ Unification itself is not the goal, but the benefits after unification is the lo
 
 - Company value: Unify technology stack, reduce company operation cost, reduce the difficulty of transitioning to microservices and cloud-native, and accelerate enterprise digital transformation.
 
-Last but not least is the APISIX [Slack channel](https://apisix.apache.org/docs/general/community#slack), any questions can be left here or on [Github issue](https://github.com/apache/apisix/issues), there will be someone to respond quickly, thanks again.
+Last but not least is the APISIX [Slack channel](https://apisix.apache.org/docs/general/join#slack), any questions can be left here or on [Github issue](https://github.com/apache/apisix/issues), there will be someone to respond quickly, thanks again.
```

Comment:
Seems we should use `#join-the-slack-channel` instead of `#slack` here, right ? :)

Reply:
The slash is too long IMO 😅 and `join slack` or `community slack` is semantic enough 😉

Reply:
I see, after this change https://github.com/apache/apisix-website/pull/974/files#diff-6d85625f2465cb8ff2ebda28c9d6ac33b9930a4988b9be0510d47376665c1585R30, @navendu-pottekkat you could also update Heading IDs :)

Reply:
Yes. I missed that. Will update those links.